components/features/features-content.tsx

```diff
@@ -95,10 +95,10 @@ export const hiddenContent: FeatureProps[] = [
+          'https://carbon.now.sh/embed?bg=rgba%28187%2C187%2C187%2C0%29&t=seti&wt=sharp&l=python&width=660&ds=false&dsyoff=3px&dsblur=13px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=import%2520pandas%2520as%2520pd%250Afrom%2520kedro.pipeline%2520import%2520pipeline%252C%2520node%250A%250A%250Adef%2520clean_data%28weather_df%253A%2520pd.DataFrame%29%2520-%253E%2520pd.DataFrame%253A%250A%2520%2520%2520%2520clean_weather_df%2520%253D%2520weather_df.drop%28%2522temperature%2522%252C%2520axis%253D1%29%250A%2520%2520%2520%2520return%2520clean_weather_df%250A%250A%250Adef%2520join_meteoro_data%28%250A%2520%2520%2520%2520weather_df%253A%2520pd.DataFrame%252C%2520climate_df%253A%2520pd.DataFrame%250A%29%2520-%253E%2520pd.DataFrame%253A%250A%2520%2520%2520%2520weather_climate_df%2520%253D%2520weather_df.merge%28%250A%2520%2520%2520%2520%2520%2520%2520%2520climate_df%252C%2520on%253D%255B%2522country_code%2522%252C%2520%2522area_code%2522%255D%252C%2520how%253D%2522inner%2522%250A%2520%2520%2520%2520%29%250A%2520%2520%2520%2520return%2520weather_climate_df%250A%250A%250Adata_processing_pipeline%2520%253D%2520pipeline%28%250A%2520%2520%2520%2520%255B%250A%2520%2520%2520%2520%2520%2520%2520%2520node%28%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520func%253Dclean_data%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520inputs%253D%2522weather_data%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520outputs%253D%2522clean_weather_data%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%29%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520node%28%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520func%253Djoin_meteoro_data%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520inputs%253D%255B%2522clean_weather_data%2522%252C%2520%2522climate_change_data%2522%255D%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520outputs%253D%2522weather_climate_data%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%29%252C%250A%2520%2520%2520%2520%255D%250A%29',
         style: {
           border: 0,
-          height: '570px',
+          height: '650px',
```

Comment:
Can you please make this taller so the user doesn't have the ability to scroll within the code sample? I don't know what the value should be. You may have to play around with it on the deploy preview link.

```suggestion
          height: '700px',
```

Reply:
Hmm... I actually looked up the size of the image and set it to that, tested in deploy preview, and shared the screenshot--and I didn't have to scroll. I bumped it earlier because I did have to.

Maybe I can just add a 10% buffer, if there's no "better" way to figure out the right answer.
```suggestion
          height: '715px',
```

Reply:
Try it.

Reply:
@tynandebold Preview still looks good to me, but I didn't have any problem even initially on my device. Do you want to check that it looks fine for you?